### DISCUSSION TITLE
Update output for help text formatting to not use '='

### DISCUSSION THREAD

**Discussion by s2k** [erikhuda/thor#839] [Ideas]
🌈 (Not a pull request, but still)

## Current Behaviour

Thor apps, such as `rails` display options that come with an argument with an equals sign.
In the example below, that's `[--ruby=PATH]` 

```
> rails help new
Usage:
  rails new APP_PATH [options]

Options:
…
  -r, [--ruby=PATH]               # Path to the Ruby binary of your choice
```

## Suggested Change 

Use a space instead of the '=' in the help output, like this

```
  -r, [--ruby PATH]               # Path to the Ruby binary of your choice
```

## Rationale

It seems to be common practice to not use the '=' sign, neither in practical usage, nor documentation.

Examples:

* The "Agile Web Development with Rails" books (the beta for Rails 7, in particular)
* @dhh on Twitter ➙ https://twitter.com/dhh/status/1472002099684659202
* The Rails team in the announcement of Rails 7 at https://rubyonrails.org/2021/12/15/Rails-7-fulfilling-a-vision
* The `tailwindcss-rails` documentation at https://github.com/rails/tailwindcss-rails#installation

### Note

I'd be happy to work on & provide a corresponding PR, should I get the specs to pass.

**Comment by dorner** [comment]
I think it depends on the tool. This Stackoverflow, for example, indicates that both styles are used in various contexts: https://unix.stackexchange.com/questions/573377/do-command-line-options-take-an-equals-sign-between-option-name-and-value

**Comment by rafaelfranca** [comment]
I think this would be a nice addition but I think it should be configurable per command. Do you still want to work on it?
